packages/gatsby-theme-docs/src/components/multi-language-code-examples.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ContentNotifications,
+  MultiCodeBlock,
+  CodeBlock,
+} from '@commercetools-docs/ui-kit';
+import useCodeExamples from '../hooks/use-code-examples';
+
+function MultiLanguageCodeExamples(props) {
+  const codeExamples = useCodeExamples();
+
+  try {
+    const codeBlockProps = extractProps(props.children, codeExamples);
+
+    return (
+      <MultiCodeBlock title={props.title}>
+        {codeBlockProps.map(cbProps => (
+          <CodeBlock
+            key={cbProps.language}
+            content={cbProps.content}
+            language={cbProps.language}
+            highlightLines={cbProps.highlightLines || []}
+            noPromptLines={cbProps.noPromptLines || []}
+          />
+        ))}
+      </MultiCodeBlock>
+    );
+  } catch (e) {
+    if (process.env.NODE_ENV !== 'production') {
+      return (
+        <ContentNotifications.Error>{e.message}</ContentNotifications.Error>
+      );
+    }
+
+    throw new Error(e.message);
```

Comment:
We should throw the error as-is.

```suggestion
    throw e;
```